app/frontend/src/features/messages/surfing/SurfingTab.tsx

```diff
@@ -25,36 +24,39 @@ export default function SurfingTab({
   type: "all" | "hosting" | "surfing";
   onlyActive?: boolean;
 }) {
-  ///TODO: add url chat opening
-  const [hostRequest, setHostRequest] = useState<HostRequest.AsObject | null>(
-    null
-  );
   const { data: hostRequests, isLoading, error } = useQuery<
     HostRequest.AsObject[],
     GrpcError
   >(["hostRequests", type, onlyActive], () =>
```

Comment:
`onlyActive` isn't in the frontend yet, but it is a backend parameter. The query results when it is true are not valid for when it is false

Reply:
Right, is that to do with request filtering we are implementing later?

Reply:
Yeah

Reply:
I wonder if it's worth being more explicit in the query keys for things that aren't `[entity, id]` so it's clearer what it's meant to do. So `["hostRequests", { type, onlyActive }]` instead of what we got. Probably doesn't look that much different in code, but could be useful for debugging with the React Query dev tool.